platform/gtk4/cog-platform-gtk4.c

```diff
@@ -41,8 +41,12 @@ G_DEFINE_DYNAMIC_TYPE_EXTENDED(
     g_io_extension_point_implement(COG_MODULES_PLATFORM_EXTENSION_POINT, g_define_type_id, "gtk4", 400);)
 
 /*
- * TODO
- * - multi-views
+ * TODO:
+ * - Multiple views.
+ * - Call cog_gl_renderer_finalize() when GL area is being destroyed.
+ * - Implement an actual widget that wraps exported the EGLImages into a
```

Comment:
extraneous 'the' after 'exported'

Reply:
The “the“ shall be removed before merging, indeed.